apps/backend/src/policies.ts

```diff
@@ -0,0 +1,58 @@
+export const policies = {
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: [`'self'`],
+      imgSrc: `'self' data:`,
+      scriptSrc: [`'self'`],
+      styleSrc: [
+        `'self'`,
+        `'sha256-/jDKvbQ8cdux+c5epDIqkjHbXDaIY8RucT1PmAe8FG4='`,
+        `'sha256-ezdv1bOGcoOD7FKudKN0Y2Mb763O6qVtM8LT2mtanIU='`,
+        `'sha256-eaPyLWVdqMc60xuz5bTp2yBRgVpQSoUggte1+40ONPU='`,
+      ],
+      fontSrc: [`'self'`],
+    },
+  },
+  permissionsPolicy: `'accelerometer=()',
```

Comment:
This is to address a low sev warning, correct? Do we need all of these to be ()?